pandas/io/stata.py

```diff
@@ -585,25 +570,21 @@ def _cast_to_stata_types(data):
             else:
                 data[col] = data[col].astype(np.float64)
                 if data[col].max() >= 2 ** 53 or data[col].min() <= -(2 ** 53):
-                    ws = precision_loss_doc % ("int64", "float64")
+                    ws = precision_loss_doc.format("int64", "float64")
         elif dtype in (np.float32, np.float64):
             value = data[col].max()
             if np.isinf(value):
                 raise ValueError(
-                    "Column {col} has a maximum value of "
-                    "infinity which is outside the range "
-                    "supported by Stata.".format(col=col)
+                    f"Column {col} has a maximum value of infinity which is outside "
+                    f"the range supported by Stata."
```

Comment:
```suggestion
                    "the range supported by Stata."
```

Don't need the prefix unless actually substituting values

Reply:
Removed or converted to multiline